core/ui/Actions/new-journal.tid

```diff
@@ -3,7 +3,7 @@ tags: $:/tags/Actions
 description: create a new journal tiddler
 
 \whitespace trim
-\function tf.get-tags() [<textFieldTags>] [<tagsFieldTags>] +[join[]]
+\function tf.get-tags() [<textFieldTags>] [<tagsFieldTags>] +[join[ ]]
```

Comment:
I did test is without the "space" in the join and there was a problem. So the `+[join[ ]]` -- **is needed**  👍